target/scripts/start-mailserver.sh

```diff
@@ -41,12 +44,17 @@ VARS[FAIL2BAN_BLOCKTYPE]="${FAIL2BAN_BLOCKTYPE:=drop}"
 VARS[FETCHMAIL_PARALLEL]="${FETCHMAIL_PARALLEL:=0}"
 VARS[FETCHMAIL_POLL]="${FETCHMAIL_POLL:=300}"
 VARS[LDAP_START_TLS]="${LDAP_START_TLS:=no}"
-VARS[LOGROTATE_INTERVAL]="${LOGROTATE_INTERVAL:=${REPORT_INTERVAL:-daily}}"
+VARS[LOGROTATE_INTERVAL]="${LOGROTATE_INTERVAL:=daily}"
 VARS[LOGWATCH_INTERVAL]="${LOGWATCH_INTERVAL:=none}"
+VARS[LOGWATCH_RECIPIENT]="${LOGWATCH_RECIPIENT:=${REPORT_RECIPIENT}}"
+VARS[LOGWATCH_SENDER]="${LOGWATCH_SENDER:=${REPORT_SENDER}}"
 VARS[MOVE_SPAM_TO_JUNK]="${MOVE_SPAM_TO_JUNK:=1}"
 VARS[NETWORK_INTERFACE]="${NETWORK_INTERFACE:=eth0}"
 VARS[ONE_DIR]="${ONE_DIR:=1}"
 VARS[OVERRIDE_HOSTNAME]="${OVERRIDE_HOSTNAME}"
```

Comment:
If this line ever receives a fallback value, it will be ignored by `_obtain_hostname_and_domainname` call that has now been moved prior to the declaration/assignment.

Reply:
Which line exactly do you mean? `VARS[OVERRIDE_HOSTNAME]="${OVERRIDE_HOSTNAME}"`? I cannot identify a line that would be negatively impacted. I spotted a missing `:-` though for good style.

Reply:
Now I see what you're referring to. I think this is fine as we can check for fallback again in `_obtain_hostname_and_domainname` if the variable is not set.